config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to null, not []. Setting it to [] will send the default list
```

Comment:
What we really need is a way to know if the user explicitly set this value or not. If they did, we use it as-is; if they didn't we use our default. I don't know how to accomplish this with Joi.